services/app-web/src/pages/SubmissionWithdraw/SubmissionWithdraw.tsx

```diff
@@ -31,6 +46,45 @@ const submissionWithdrawSchema = Yup.object().shape({
     ),
 })
 
+export const shouldWithdraw = (
```

Comment:
suggestion: this is actually storing value for `shouldWarnOnWithdraw` it doesnt prevent withdraw